src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,57 @@
+.nav {
+
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__item:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: $header-height;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 500;
+    color: $header-link-text-color;
+    transition: color $header-transition-duration;
+
+    /* LoVeHAte */
+    &:hover {
+      color: $header-link-text-hover-color;
+    }
+
+    &:active::after {
```

Comment:
I think you don`t need it cause we have no case in this task when the link will be active, so the main aim it`s to set the class is-active too different nav-link and make them with underline, and that's enough.
The `:active` pseudo-class will work only in case when holding your mouse LMB on the link but here we don`t need it.
![image](https://github.com/mate-academy/layout_catalog/assets/64101898/38bf8ca9-a1fb-48f3-8b25-6264df0fdc3f)